packages/block-editor/src/components/inserter/menu.js

```diff
@@ -32,7 +32,6 @@ import InserterSearchResults from './search-results';
 import useInsertionPoint from './hooks/use-insertion-point';
 import { store as blockEditorStore } from '../../store';
 import TabbedSidebar from '../tabbed-sidebar';
-import { useZoomOut } from '../../hooks/use-zoom-out';
```

Comment:
Do we use this hook somewhere else? Should we remove it?

Reply:
We use it in global styles too